openedx/features/enterprise_support/enrollments/utils.py

```diff
@@ -8,7 +8,11 @@
 
 from common.djangoapps.student.models import User
 from openedx.core.djangoapps.enrollments import api as enrollment_api
-from openedx.core.djangoapps.enrollments.errors import CourseEnrollmentError, CourseEnrollmentExistsError
+from openedx.core.djangoapps.enrollments.errors import (
+    CourseEnrollmentError,
+    CourseEnrollmentExistsError,
+    EnrollmentNotUpdatableError,
```

Comment:
tiny nit / naming suggestion: I wonder if (for slightly more consistency in naming), this import should be named `CourseEnrollmentNotUpdatableError`.